### ISSUE TITLE
Install error, had to edit bower.json

### ISSUE THREAD

**Issue by walter** [simplereach/ember-cli-betamax#2]
Hi,

First, it's cool to see ember-cli-betamax. Nice work. Thanks for bringing the convenience of VCR to Ember.js.

I'm currently running ember-cli 0.1.15 and when I ran through the install steps, ember server complained as follows:

```
$ ember s
ember s
version: 0.1.15
Livereload server on port 35729
Serving on http://0.0.0.0:4200/
Path or pattern "bower_components/sinon/index.js" did not match any files
Error: Path or pattern "bower_components/sinon/index.js" did not match any files
    at Object.multiGlob (/Users/walter/Development/javascript/address/node_modules/ember-cli/node_modules/broccoli-sourcemap-concat/node_modules/broccoli-kitchen-sink-helpers/index.js:202:13)
    at Class.module.exports.CachingWriter.extend.addFiles (/Users/walter/Development/javascript/address/node_modules/ember-cli/node_modules/broccoli-sourcemap-concat/concat-with-maps.js:74:13)
    at Class.module.exports.CachingWriter.extend.updateCache (/Users/walter/Development/javascript/address/node_modules/ember-cli/node_modules/broccoli-sourcemap-concat/concat-with-maps.js:52:12)
    at /Users/walter/Development/javascript/address/node_modules/ember-cli/node_modules/broccoli-sourcemap-concat/node_modules/broccoli-caching-writer/index.js:92:34
    at lib$rsvp$$internal$$tryCatch (/Users/walter/Development/javascript/address/node_modules/ember-cli/node_modules/promise-map-series/node_modules/rsvp/dist/rsvp.js:489:16)
    at lib$rsvp$$internal$$invokeCallback (/Users/walter/Development/javascript/address/node_modules/ember-cli/node_modules/promise-map-series/node_modules/rsvp/dist/rsvp.js:501:17)
    at lib$rsvp$$internal$$publish (/Users/walter/Development/javascript/address/node_modules/ember-cli/node_modules/promise-map-series/node_modules/rsvp/dist/rsvp.js:472:11)
    at Object.lib$rsvp$asap$$flush [as _onImmediate] (/Users/walter/Development/javascript/address/node_modules/ember-cli/node_modules/promise-map-series/node_modules/rsvp/dist/rsvp.js:1290:9)
    at processImmediate [as _immediateCallback] (timers.js:354:15)
```

I tried removing bower_components, etc., but what made things work for me was to edit bower.son like so:

```
    "sinon-1.12.2": "http://sinonjs.org/releases/sinon-1.12.2.js",
```

to

```
    "sinon": "http://sinonjs.org/releases/sinon-1.12.2.js",
```

After that, I rm'd tmp/\* and ember server started up without complaint.

**Comment by walter** [comment]
Must have been my environment.

I installed an addon that updated bower.json and reverted my change (i.e. "sinon" was reverted to "sinon-1.12.1"), but ember serve is not complaining.

So going ahead and closing this. Sorry for the false alarm.

**Comment by walter** [comment]
Spoke to soon. When I installed something else that had a simon dependency, it installed "sinon-1.12.1" in _addition_ to the "sinon" in bower.json that I had added previously.

If I remove this:

``` javascript
    "sinon": "http://sinonjs.org/releases/sinon-1.12.2.js",
```

I get the original issue when running ember serve.

**Comment by aquamme** [comment]
I can replicate this issue. My workaround was to symlink the sinon-1.12 folder to sinon in bower_components, but of course that isn't a long-term fix.

**Comment by andremalan** [comment]
Should be fixed in a9bd43a25a196d3db836b641f8ce9ba44496366f. You can run "ember generate ember-cli-betamax" again to update bower. Let me know if it works.
